keys/keys_test.go

```diff
@@ -75,6 +76,25 @@ func TestKeyAddress(t *testing.T) {
 	}
 }
 
+func TestMakeTableIndexKey(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	key := MakeTableIndexKey(12, 345, []byte("foo"), []byte("bar"))
+	expKey := MakeKey(TableDataPrefix, encoding.EncodeUvarint(nil, 12), encoding.EncodeUvarint(nil, 345), encoding.EncodeBytes(nil, []byte("foo")), encoding.EncodeBytes(nil, []byte("bar")))
```

Comment:
Added in more unittest coverage and pretty much stole what you recommended. Thanks